myvoice/templates/clinics/report.html

```diff
@@ -279,32 +279,58 @@ <h2 style="position: absolute; top: 7px; left: 15px;">PARTICIPATION BY FACILITY<
       </div>
```

Comment:
Another template myvoice/templates/clinics/report_response.html needs to change as well to reflect the new design.